backend/src/common/exception/error.exception.ts

```diff
@@ -0,0 +1,14 @@
+import { HttpException, HttpStatus } from '@nestjs/common';
+
+export class ErrorException extends HttpException {
+  constructor(message: string, code: number, status: HttpStatus) {
+    super(
+      {
+        message,
+        code,
+        timestamp: new Date().toISOString(),
```

Comment:
```suggestion
        timestamp: new Date().toLocaleString();
```
이렇게 하면 타임존이 적용된 시간을 얻을 수 있습니다!

Reply:
수정하겠습니다 👍🏻